### DISCUSSION TITLE
Two level grouping in tables

### DISCUSSION THREAD

**Discussion by lsatolli** [filamentphp/filament#9147] [Ideas]
is there any plan to create 2 level grouping for table rows? 

i.e. orders table that can be grouped by customer name and address or by status and day

**Comment by michaelklopf** [comment]
Maybe this plugin can help you https://filamentphp.com/plugins/kenneth-sese-advanced-tables

**Reply by lsatolli** [reply]
I know it. it is a very awesome plugin but it does not have 2 levels aggregation I mean someting like a table pivot in excel in which rows can be open an close on more than 1 level
